tests/elm-package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "1.0.0",
+    "summary": "Sample Elm Test",
+    "repository": "https://github.com/user/project.git",
+    "license": "BSD-3-Clause",
+    "source-directories": [
+        ".",
+        "../src"
```

Comment:
if you're going to do this, then you need to have `"native-modules": true` in this elm-package.